src/pages/ContestPage.tsx

```diff
@@ -58,7 +59,7 @@ const ContestPage = (props: RouteComponentProps<ContestParams>) => {
   // - confirm intent when navigating away without selecting a candidate
 
   return (
-    <Screen>
+    <Screen ref={rootRef as any}>
```

Comment:
All changes in this file are unnecessary and represent a path I did not end up taking. I was trying to get a reference to the root element of the screen for the purpose of adding/removing event handlers, but I ended up doing that by adding handlers to `FocusManager`. I'll remove these.